### DISCUSSION TITLE
No data in Traefik dashboard

### DISCUSSION THREAD

**Discussion by ilanKushnir** [carlosedp/cluster-monitoring#133] [Q&A]
Hi all!

[SCREENSHOT](https://imgur.com/a/0PoVBbO)
There is no data In my `Traefik` dashboard.
I changed it to `true` in `values.jsonnet` file and still no data...
do you have any idea why? should I deploy some additional exporter for that?

I'm using it on a cluster of 2 Raspberry PIs 4 (8GB) - running K3S with the default Traefik infress controller

thanks!!!

**Comment by cyc1ingsir** [comment]
Have you tried to run the queries defined for the panels from the Prometheus UI?
I noticed that some dashboards won't display data any longer if you used a newer version of K3s as the metrics have been renamed.

**Reply by ilanKushnir** [reply]
exactly, I think the metrics had gone when K3S upgraded Traefik to v.2
couldn't find any metrics in Prometheus :(

**Reply by tonysurma** [reply]
I have similar and the traefik targets are empty in prometheus UI and same in the service discovery.  I am wondering if something is different/new in recent k3s with traefik v2.  I have spent a few hours and I cannot get anything to work here.  I can find the metrics manually hitting urls etc so i feel like they are there but not being exposed/found by prometheus.

**Comment by carlosedp** [comment]
The config here is for Traefik 1.7. I would accept a PR adding support for Traefik 2 since I don't have time to add it now.
